md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BatchAPI.java

```diff
@@ -68,11 +68,10 @@ RestWrapper get(
                     batch.setSourceInstanceExecId(jpaBatch.getInstanceExec().getInstanceExecId());
                 }
             }
-            // batch = s.selectOne("call_procedures.GetBatch", batch);
-
             restWrapper = new RestWrapper(batch, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + batchId + " selected from Batch by User:" + principal.getName());
+            LOGGER.info(RECORDWITHID + batchId + " selected from Batch by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)